packages/components/src/card/card-divider/component.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * Internal dependencies
+ */
+import { contextConnect } from '../../ui/context';
+import { Divider } from '../../divider';
+import { useCardDivider } from './hook';
+
+/**
+ * @param {import('../../ui/context').PolymorphicComponentProps<import('../../divider').DividerProps, 'hr'>} props
+ * @param {import('react').Ref<any>} forwardedRef
+ */
+function CardDivider( props, forwardedRef ) {
+	const dividerProps = useCardDivider( props );
+
+	return <Divider { ...dividerProps } ref={ forwardedRef } />;
+}
+
+/**
+ * CardDivider renders an optional divider within a `<Card />
```

Comment:
We should add that it is mainly used to divide `CardBodies` from each other?

Reply:
Good point! Added in https://github.com/WordPress/gutenberg/pull/32243/commits/cb08d29eda202cfa36b35530fe428647247a23b0